seta-react/src/pages/CommunitiesPage/components/Sidebar/MembersList/MembersList.tsx

```diff
@@ -45,50 +31,50 @@ const useStyles = createStyles(theme => ({
 const MembersList = () => {
   const { classes, cx } = useStyles()
   const [scrolled, setScrolled] = useState(false)
-  const theme = useMantineTheme()
-  const { memberships, getMembershipRequests } = useNotifications()
+  // const theme = useMantineTheme()
+  // const { memberships, getMembershipRequests } = useNotifications()
 
-  useEffect(() => {
-    getMembershipRequests()
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
+  // useEffect(() => {
+  //   getMembershipRequests()
+  //   // eslint-disable-next-line react-hooks/exhaustive-deps
+  // }, [])
 
-  if (memberships) {
-    if (memberships.length === 0) {
-      return <ComponentEmpty />
-    }
-  }
+  // if (memberships) {
+  //   if (memberships.length === 0) {
+  //     return <ComponentEmpty />
+  //   }
+  // }
 
-  if (!memberships) {
-    return <ComponentLoading />
-  }
+  // if (!memberships) {
+  //   return <ComponentLoading />
+  // }
 
   const rows =
-    memberships && memberships?.length > 0
-      ? memberships?.map(row => (
-          <tr key={row.community_id}>
-            <td>{row.community_id.charAt(0).toUpperCase() + row?.community_id.slice(1)}</td>
-            <td>
-              <Badge
-                color={statusColors[row.status.toLowerCase()]}
-                variant={theme.colorScheme === 'dark' ? 'light' : 'outline'}
-              >
-                {row.status.toUpperCase()}
-              </Badge>
-            </td>
-            <td>{row.message.charAt(0).toUpperCase() + row?.message.slice(1)}</td>
-            <td>{new Date(row.initiated_date).toDateString()}</td>
-            <td>{row.requested_by_info?.full_name}</td>
-            <td>{row.review_date ? new Date(row.review_date).toDateString() : null}</td>
-            <td>{row.reviewed_by_info?.full_name}</td>
-            <td>
-              <Group spacing={0}>
-                <UpdateMemberRequest props={row} />
-              </Group>
-            </td>
-          </tr>
-        ))
-      : []
+    // memberships && memberships?.length > 0
```

Comment:
Is there a reason for commenting out all this code instead of removing it?

Reply:
I wanted to fix it for the moment and review the code to get it working properly. I removed some of the commented code, I just kept the part that I will reuse with the new API.
In the end I won't have commented code

Reply:
I don't think we should ever merge commented code to development/master/main.
If you need to keep track of it for some future work, you can stash it and then remove it from the PR branch.

It's easy to forget about commented code and the principal branch should always be clean and nice, without work-in-progress code.